data/ecosystem/vendors.yaml

```diff
@@ -28,6 +28,12 @@
   contact:
   oss: false
   commercial: true
+- name: OpenSearch
+  nativeOTLP: true
+  url: https://opensearch-project.github.io
+  contact: https://opensearch.org/connect.html
```

Comment:
Replace this with an email address or github handle, this is for our usage, if anything breaks and we need to reach out to the project to let them know.